src/cordova/plugin/remove.js

```diff
@@ -68,6 +68,7 @@ function remove (projectRoot, targets, hooksRunner, opts) {
                                 // If platform does not returned anything we'll need
                                 // to trigger a prepare after all plugins installed
                                 // TODO: if didPrepare is falsy, what does that imply? WHY are we doing this?
+                                events.emit('verbose', 'Calling prepare.');
```

Comment:
I think you might want to remove this event. Since Prepare will only be called if `didPrepare` is falsy. So you could get the output `calling prepare` when you actually don't call prepare